pkgs/development/libraries/libgpg-error/fix-1.42-cross-compilation.patch

```diff
@@ -0,0 +1,142 @@
+diff --git a/src/gen-lock-obj.sh b/src/gen-lock-obj.sh
```

Comment:
Why is this patch so long for a one line diff?

Reply:
It's the exact patch file I linked, off their phabricator. Decided not to minimise it, as if the base code changes in future, it will incorporate the fix, so the patch should be deleted anywhere.